federatedscope/core/configs/cfg_fl_setting.py

```diff
@@ -49,7 +49,12 @@ def extend_fl_setting_cfg(cfg):
     cfg.distribute.client_port = 50050
     cfg.distribute.role = 'client'
     cfg.distribute.data_file = 'data'
```

Comment:
The data-related keyword `distribute.data_file` in distributed mode seems to be the same meaning as `data.root`.

Reply:
‘data.root’ is a file directory and `distribute.data_file` includes the filename here